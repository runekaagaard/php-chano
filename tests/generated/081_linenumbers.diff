001- 1. line 1
001+ Fatal error: Call to undefined method DtlIter::linenumbers() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/081_linenumbers.php on line 5
002- 2. line 2