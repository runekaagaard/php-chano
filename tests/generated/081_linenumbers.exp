1. line 1
2. line 2