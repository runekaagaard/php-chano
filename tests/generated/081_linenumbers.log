
---- EXPECTED OUTPUT
1. line 1
2. line 2
---- ACTUAL OUTPUT
Fatal error: Call to undefined method DtlIter::linenumbers() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/081_linenumbers.php on line 5
---- FAILED
