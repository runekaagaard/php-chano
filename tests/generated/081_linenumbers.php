<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => 'line 1
line 2')));
foreach ($items as $i) echo $i->input->linenumbers();
