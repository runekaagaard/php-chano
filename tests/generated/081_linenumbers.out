Fatal error: Call to undefined method DtlIter::linenumbers() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/081_linenumbers.php on line 5